docs/PRIORITIES.md

```diff
@@ -1,46 +1,35 @@
 # Post-0.25/GoS Pre-Release
 
-## Staking/Slashing/Stability
-
-- Other slashing issues blocking for launch - [#1256](https://github.com/cosmos/cosmos-sdk/issues/1256)
-- Miscellaneous minor staking issues
-  - [List here](https://github.com/cosmos/cosmos-sdk/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astaking+label%3Aprelaunch)
-  - Need to figure out scope of work here to estimate time
-  - @rigelrozanski to start next
-- Consider "tombstone" / "prison" - double-sign and you can never validate again - https://github.com/cosmos/cosmos-sdk/issues/2363
-
 ## Multisig
 
 - Need to test changes in https://github.com/cosmos/cosmos-sdk/pull/2165
 - Spam prevention - https://github.com/cosmos/cosmos-sdk/issues/2019
 
 ## ABCI Changes
 
-- Need to update for new ABCI changes when/if they land - error string, tags are list of lists
-- Need to verify correct proposer reward semantics
```

Comment:
I don't think we've done this either

Reply:
I'll add this back in, however, I thought from conversations with @jaekwon that this was complete.